package.json

```diff
@@ -11,11 +11,19 @@
     "lint": "yarn run lint:check --fix"
   },
   "dependencies": {
+    "emotion": "^10.0.9",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "react-scripts-ts": "3.1.0"
+    "react-redux": "^6.0.1",
+    "react-scripts-ts": "3.1.0",
+    "redux": "^4.0.1",
+    "redux-actions": "^2.6.5"
   },
   "devDependencies": {
+    "@emotion/core": "^10.0.9",
```

Comment:
`@emotion` dependencies need to be in `dependencies` and **not** `devDependencies`

Reply:
Change has been made and re-pushed.